libs/designer-ui/src/lib/editor/base/plugins/TokenTypeahead.tsx

```diff
@@ -66,13 +67,14 @@ function TokenMenuItem({
 }
 
 interface TokenTypeAheadPluginProps {
-  openTokenPicker: (tokenPickerMode: TokenPickerMode) => void;
   isEditorFocused?: boolean;
-  hideExpression?: boolean;
+  hideTokenpickerOptions?: hideButtonOptions;
```

Comment:
Just for consistent capitalization
```suggestion
  hideTokenPickerOptions?: hideButtonOptions;
```

Reply:
rip nevermind I was too slow

Reply:
oops, ill make this change in my next PR